config/routes.rb

```diff
@@ -207,7 +207,17 @@
         end
       end
 
-      resources :peer_reviews, only: :index do
+      resources :results, only: [:edit], path: '/peer_reviews' do
```

Comment:
Scoping was giving me issues, and at best it was putting it beforehand (ex: `assignments/peer_reviews/1/results/5`) whereas this way does it as per requested on slack (meaning: `assignments/1/peer_reviews/results/5/edit`)